experiment/simulation/js/main.js

```diff
@@ -1 +1,148 @@
-//Your JavaScript goes in here
+import data from "./data.js";
+
+const setMolecule = (angle, animate = false) => {
+  let width = 400;
+  if (window.innerWidth < 900) {
+    width = window.innerWidth;
+  }
+  let movie = new ChemDoodle.MovieCanvas3D("movie", width, 400);
+  if (data[angle] != null) {
+    if (animate) {
+      for (let angle = -180; angle <= 180; angle += 3) {
+        movie.addFrame([ChemDoodle.readXYZ(data[angle]["geometry"])], []);
+      }
+    } else {
+      movie.addFrame([ChemDoodle.readXYZ(data[angle]["geometry"])], []);
+    }
+
+    movie.styles.set3DRepresentation("Ball and Stick");
+    movie.styles.atoms_displayLabels_3D = true;
+    movie.styles.backgroundColor = "transparent";
+    movie.loadMolecule(movie.frames[0].mols[0]);
+    if (animate) {
+      movie.startAnimation();
+    }
+  }
+  return movie;
+};
+
+const initChart = () => {
+  let xyValues = Object.keys(data).map((key) => ({
+    x: key,
+    y: data[key]["energy"],
+  }));
+
+  return new Chart("myChart", {
+    type: "scatter",
+    data: {
+      datasets: [
+        {
+          pointRadius: 4,
+          pointBackgroundColor: "rgb(0,0,255)",
+          pointBorderColor: "rgb(133, 193, 233)",
+          data: xyValues,
+        },
+      ],
+    },
+    options: {
+      legend: { display: false },
+      scales: {
+        xAxes: [
+          {
+            scaleLabel: {
+              display: true,
+              labelString: "Angle (in degree)",
+            },
+          },
+        ],
+
+        yAxes: [
+          {
+            scaleLabel: {
+              display: true,
+              labelString: "Energy (Ha)",
+            },
+          },
+        ],
+      },
+    },
+  });
+};
+
+const highlightChart = (angle) => {
+  chart.data.datasets[0].pointBackgroundColor = [];
+  chart.data.datasets[0].pointRadius = [];
+  for (var i = 0; i < chart.data.datasets[0].data.length; i++) {
+    if (chart.data.datasets[0].data[i]["x"] == angle) {
```

Comment:
Always use === for equality and !== for inequality unless you have a valid use case for implicit data type conversion